jina/types/message/__init__.py

```diff
@@ -345,28 +344,6 @@ def _pod_str(r):
         route_str.append('⚐')
         return colored('▸', 'green').join(route_str)
 
-    def add_route(self, name: str, identity: str):
-        """Add a route to the envelope.
-
-        :param name: the name of the pod service
-        :param identity: the identity of the pod service
-        """
-        self._add_route(name, identity, self.envelope)
```

Comment:
Why do we remove this?

Reply:
Understood. I would do it in another PR however. I am not sure if in the future for monitoring may be useful.

But that is far from a discussion here

Reply:
yes it doesnt really belong here, but the issue is that the `routes` are already removed from the envelope (its still in the request so we can send it back to the client from the gateway if so desired).
Being removed from the envelope proto means that constructing a Message fails at the moment, which prevents any test etc. Thats why I fixed it here. I can also do it in a separate PR, merge that one and rebase this, but thats quite a detour and I think will happen all the time now with this big refactoring?